src/reducers/sponsors/sponsor-page-forms-list-reducer.js

```diff
@@ -0,0 +1,301 @@
+          ? epochToMomentTimeZone(a.expires_at, state.summitTZ)?.format(
+              "YYYY/MM/DD"
+            )
+          : "N/A";
+
+        return {
+          id: a.id,
+          code: a.code,
+          name: a.name,
+          items_count: a.items_count || 0,
+          allowed_add_ons: a.allowed_add_ons,
+          is_archived: a.is_archived,
+          opens_at: opensAt,
+          expires_at: expiresAt
+        };
+      });
+
+      return {
+        ...state,
+        customizedForms: {
+          ...state.customizedForms,
+          forms,
+          currentPage,
+          totalCount: total,
+          lastPage
+        }
+      };
+    }
+    case SPONSOR_MANAGED_FORMS_ADDED: {
+      const newForm = payload.response;
+
+      newForm.opens_at = payload.response.opens_at
+        ? epochToMomentTimeZone(
+            payload.response.opens_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+      newForm.expires_at = payload.response.expires_at
+        ? epochToMomentTimeZone(
+            payload.response.expires_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+
+      return {
+        ...state,
+        managedForms: {
+          ...state.managedForms,
+          forms: [...state.managedForms.forms, newForm],
+          totalCount: state.totalCount + 1
+        }
+      };
+    }
+    case SPONSOR_CUSTOMIZED_FORM_ADDED: {
+      const newForm = payload.response;
+
+      newForm.opens_at = payload.response.opens_at
+        ? epochToMomentTimeZone(
+            payload.response.opens_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+      newForm.expires_at = payload.response.expires_at
+        ? epochToMomentTimeZone(
+            payload.response.expires_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+
+      return {
+        ...state,
+        customizedForms: {
+          ...state.customizedForms,
+          forms: [...state.customizedForms.forms, newForm],
+          totalCount: state.totalCount + 1
+        }
+      };
+    }
+    case SPONSOR_CUSTOMIZED_FORM_UPDATED: {
+      const newForm = payload.response;
+
+      newForm.opens_at = payload.response.opens_at
+        ? epochToMomentTimeZone(
+            payload.response.opens_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+      newForm.expires_at = payload.response.expires_at
+        ? epochToMomentTimeZone(
+            payload.response.expires_at,
+            state.summitTZ
+          )?.format("YYYY/MM/DD")
+        : "N/A";
+
+      const forms = state.customizedForms.forms.map((form) => {
+        if (form.id === newForm.id) {
+          return newForm;
+        }
+        return form;
+      });
+
+      return {
+        ...state,
+        customizedForms: {
+          ...state.customizedForms,
+          forms,
+          totalCount: state.totalCount + 1
```

Comment:
@santipalenque do we really need to increase the count here ? its updating the form so the form already exists also it does not seem to be the rite state variable